### DISCUSSION TITLE
Paradise Lost

### DISCUSSION THREAD

**Discussion by Groupix05** [DynamicTreesTeam/DynamicTrees#941] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/paradise-lost
For Minecraft 1.20.1

Trees :
- Lavendar Wisteria
- Rose Wisteria
- Frost Wisteria
- Mottled Aurel
- Aurel

**Comment by supermassimo** [comment]
don't forget to include the curseforge link :)
